docs/content/theme/color-palette.md

```diff
@@ -18,7 +18,11 @@ tags = [ "theme" ]
 
 ## Text & links
 
-Text and links are the most fundamental pieces of any web property, so they have their own variables aside from other UI elements. They also respond to context, such as a bit of text in a dark gray card would need to change to white. Or a link in a dark gray card would need to become a lighter shade of blue.  Here are a few examples:
+Text and links are the most fundamental pieces of any web property, so they have their own variables aside from other UI elements. They also respond to context, such as a bit of text in a dark gray card would need to change to white. Or a link in a dark gray card would need to become a lighter shade of blue.  
```

Comment:
```suggestion
Text and links, being fundamental pieces of any web property, have their own set of variables. 
These variables assume a light background by default, assigning typography colors that pass accessibility when placed on white or very light gray contexts.  In addition, we provide contextual variables for these typographical elements for when they exist inside a dark or saturated context.  Text in a black card for example would need to flip to white to pass accessibility standards while links on a blue or red shade would need to update to white in order to be visible.
```

I found the wording above a bit difficult to understand. What would you think of something like this? ^